lib/roger/info.ex

```diff
@@ -51,13 +51,16 @@ defmodule Roger.Info do
   requeues the message using a nack.
   """
   def queued_jobs(partition_id, queue_type, count \\ 100) do
-    {:ok, channel} = AMQPClient.open_channel()
+    connection_name = Application.get_env(:roger, :connection_name)
 
-    queue = Roger.Queue.make_name(partition_id, queue_type)
-    result = get_queue_messages(channel, queue, count)
+    with {:ok, amqp_conn} <- AMQP.Application.get_connection(connection_name),
+         {:ok, channel} <- AMQP.Channel.open(amqp_conn) do
+      queue = Roger.Queue.make_name(partition_id, queue_type)
+      result = get_queue_messages(channel, queue, count)
 
-    :ok = AMQP.Channel.close(channel)
-    result
+      AMQP.Channel.close(channel)
```

Comment:
Why is the channel closed? I think the performance-loss of having to open a new channel is greater than any CPU- / Memory usage gain.